enterprise_access/apps/content_assignments/models.py

```diff
@@ -372,6 +372,25 @@ def add_errored_expiration_action(self, exc):
             traceback=format_traceback(exc),
         )
 
+    def add_successful_redeemed_action(self):
+        """
+        Adds a successful expiration LearnerContentAssignmentAction for this assignment record.
+        """
+        return self.actions.create(
+            action_type=AssignmentActions.REDEEMED,
+            completed_at=timezone.now(),
+        )
+
+    def add_errored_redeemed_action(self, exc):
+        """
+        Adds an errored expiration LearnerContentAssignmentAction for this assignment record.
```

Comment:
nit: docstring reference expiration, but this action deals with redemption, not expiration.